index.js

```diff
@@ -99,7 +96,9 @@ module.exports = function getCardTypes(cardNumber) {
   var i, value;
   var result = [];
 
-  if (!isString(cardNumber)) { return result; }
+  if (!(typeof cardNumber === 'string' || cardNumber instanceof String)) {
```

Comment:
So `typeof cardNumber === 'string'` is not enough, and `cardNumber instanceof String` also needs to be checked?